zOSMF-Samples/workflow_sample_automation.xml

```diff
@@ -0,0 +1,189 @@
+	<workflowDescription>Sample demonstrating the use of automated steps in workflow.</workflowDescription>
+	<workflowVersion>1.0</workflowVersion>
+	<vendor>IBM</vendor>
+	<Configuration>
+		<productID>ABC123</productID>
+		<productName>Product ABC</productName>
+		<productVersion>Version 1</productVersion>
+ 	</Configuration>
+</workflowInfo>
+ 
+<!-- Declare a set of variables -->
+<variable name="st_user">
+	<label>user ID</label>
+	<abstract>User ID for the started task.</abstract>
+	<description>The user ID under whose authority the new started task will run.</description>
+	<category>Started</category>
+	<string>
+		<validationType>USERID</validationType>
+		<default>MYSTUSER</default>
+	</string>
+</variable>
+<variable name="st_group">
+	<label>group</label>
+	<abstract>Group name for the started task.</abstract>
+	<description>The group name under whose authority the started task will run.</description>
+	<category>Started</category>
+	<string>
+		<validationType>GROUP</validationType>
+		<default>SYS1</default>
+	</string>
+</variable>
+<variable name="st_gid">
+	<label>GID</label>
+	<abstract>z/OS UNIX System Services GID.</abstract>
+	<description>The UNIX group ID (GID) to be associated with the started task group.</description>
+	<category>Started</category>
+	<string>
+		<validationType>UNIXID</validationType>
+		<default>500</default>
+	</string>
+</variable>
+<!-- Declare a set of variables END -->
+ 
+<!-- This is a simple empty step which only specified as an automatable step -->
+<step name="simpleAutomatableStep" >
+	<title>Simple automatable step</title>
+	<description>This sample shows a very basic workflow automatable step.</description>
+	<instructions>Use the Instructions area to describe in detail what the user needs to do to complete the step. On completion of the step, the user should press <strong>Finish</strong> to have the step marked complete in the Workflows task. This step can be performed automatically.</instructions>
+	<weight>1</weight>
+	<autoEnable>true</autoEnable>
+</step>
+ 
+<!-- 
+	This step contains substeps and step dependencies, during automation, if the prerequisite step is complete, 
+    steps who depends on this step will become Ready automatically and then the automation will be able to continue.
+ -->
+<step name="rootStep">
+	<title>Root Step</title>
+	<description>This is a top-level step.</description>
+	<step name="subStep1">
+		<title>Second-level step 1</title>
+		<description>This is a second-level step 1.</description>
+		<step name="subStep1_1">
+			<title>Third-level step</title>
+			<description>This is a third-level step.</description>
+			<instructions>Examine this sample step.</instructions>
+			<weight>1</weight>
+			<autoEnable>true</autoEnable>
+		</step>
+	</step>
+	<step name="subStep2">
+		<title>Second-level step 2</title>
+		<description>This is a second-level step 2.</description>
+		<!-- This step take subStep1_1 as its prerequisite step. -->
```

Comment:
Gammar here seems odd.
Maybe something like
"This prereqStep element enforces that subStep1_1 must be completed before this step can be run"